### DISCUSSION TITLE
Private github repo converted to public doesn't allow Zenhub board viewing

### DISCUSSION THREAD

**Discussion by galenatjpl** [community/community#38709] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi,

  I have an issue where I created a repo initially as a private repo, and have later converted it to a public repo.
The zenhub integration doesn't work, and I think it still thinks it's a private repo. 
![Screen Shot 2022-11-09 at 11 24 47 AM](https://user-images.githubusercontent.com/47542238/200922527-b26cc6e1-757a-4c2e-82a1-4af5c4305ca3.png)

So it thinks that I need a license to view, since it would if it's a private repo.  But since it's now a public repo (i.e. using the "free" ZenHub plan), that shouldn't appear, and it should let me go straight to the board.   

I've tried everything like clearing my cookies, restarting the browser, trying incognito mode, trying a different browser (i.e. Firefox instead of Chrome), but none of these work.   This is leading me to believe that GitHub and/or Zenhub has cached some state on the server (not client-side).

As as work around, I can attach another public repo to the workspace, and set it as the "default" repo.  Then I'm able to view the zenhub board.  But this is just a hack/workaround.

The repo in question here is:  https://github.com/unity-sds/unity-on-demand

Please advise.

Thanks!

**Comment by ossanna16** [comment]
Hi there @galenatjpl and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
